packages/hoppscotch-sh-admin/src/composables/useClientHandler.ts

```diff
@@ -14,15 +14,16 @@ export function useClientHandler<
   ListItem
 >(
   query: string | TypedDocumentNode<Result, Vars> | DocumentNode,
-  getList: (result: Result) => ListItem[],
-  variables: Vars
+  variables: Vars,
+  getList?: (result: Result) => ListItem[]
```

Comment:
This breaks existing code that expects the arguments to be in the correct order.

Reply:
@JoelJacobStephen, aren't you handling these in [#3686](https://github.com/hoppscotch/hoppscotch/pull/3686/files#diff-7a00400ff4b10955eaeb86e717562fb8b11be17f9b2e0ff6c18af57765efdb1d)?